lib/gosu/compat.rb

```diff
@@ -165,7 +167,6 @@ module Gosu
   module Button; end
   
   # Channel was called SampleInstance before Gosu 0.13.0.
-  SampleInstance = Channel
```

Comment:
Thats not necessary

Reply:
[Looks like that's right](https://searchcode.com/?q=SampleInstance&lan=32), I was worried about people writing `if x.is_a? Gosu::SampleInstance`.

Reply:
This still works and triggers the deprecation warning too.